controls/button/overview.md

```diff
@@ -17,7 +17,7 @@ The Telerik UI for .NET MAUI Button control enhances the functionality of the st
 ## Key Features
 
 * [Content alignment]({% slug button-content-alignment %})&mdash;The Button enables you to control the horizontal and vertical positioning of its content.
-
+* Option for applying background image.
```

Comment:
Check the writing style of the existing content and apply it to the newly added content.